### PR TITLE
Nisus docker compose

### LINKED ISSUE #3: Docker compose
Create and prepare docker solution with containers
 - vue
 - nodejs
 - mongoDB
 - adminer(or similar)

### LINKED ISSUE #8: Add webpack config
add webpack config with correct path for import components

### LINKED ISSUE #9: Frontend docker compose
Create docker compose solution for frontend

### PR DESCRIPTION
**Pull request by stesi13**
sry zas mi nevytvorilo PR npre webpack config

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -31,4 +31,4 @@
     "sass-loader": "^8.0.2",
     "vue-template-compiler": "^2.6.11"
   }
-}
+}
```
Comment (Nephyx) [inline frontend/package.json:34]: There should be a new line at the end of the file but no biggie.

File: .devcontainer/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    command: sleep infinity
+    
+    links:
+      - mongo
+
+  mongo:
+    image: mongo
+    restart: unless-stopped
+    volumes:
+      - /data/db
```
Comment (Nephyx) [inline .devcontainer/docker-compose.yml]: I know this was generated but shouldn't there be some mapping to a local folder where our dev data would be persisted? So when I during development create some data in my local Mongo container, I'd like to have them persisted so I can use those data also a few days later when my local Mongo container is restarted or rebuilt. Check previous _docker-compose.yml_ which is deleted within this PR. There are also some other differences like `network_mode: service:db` missing in yours. And I'm not sure why those two files differ as they are both generated in the same way.

Comment (stesi13) [reply .devcontainer/docker-compose.yml]: you can create names volume and create folder in project

Comment (Nephyx) [reply .devcontainer/docker-compose.yml]: I updated the comment, please check it out again 😁

**Pull request merged**